RepoDb.Core/RepoDb/Extensions/DbCommandExtension.cs

```diff
@@ -334,6 +340,12 @@ private static void EnsureTableValueParameter(IDbDataParameter parameter)
                     classProperty?.GetDbType() ??
                     value?.GetType()?.GetDbType();
```

Comment:
The logic is correct now, so I will accept this. Though, you can also remove the additional checking and make it single line like below.
```csharp
// DbType
var dbType = (returnType != null ? clientTypeToDbTypeResolver.Resolve(returnType) : null) ??
	classProperty.GetDbType() ??
	value?.GetType()?.GetDbType() ??
        classProperty != null ? clientTypeToDbTypeResolver.Resolve(classProperty.PropertyInfo.PropertyType) : null;
```